ci-operator/step-registry/redhat-performance/benchmark-runner/redhat-performance-benchmark-runner-ref.yaml

```diff
@@ -0,0 +1,7 @@
+ref:
+  as: redhat-performance-benchmark-runner
+  from_image:
+    namespace: benchmark-runner
```

Comment:
I think we'll need to merge the config file first that will create the image and then we can use/add the step-registry files

Reply:
the namespace here will have to match [this line](https://github.com/openshift/release/pull/69278/files#diff-e200e785da56e34fca2fa37f8b3eaba318f35fdfdccdeaf7bc441515cdd58c5cR11) in the config

Reply:
Done